docker-stack.yml

```
@@ -78,6 +78,7 @@ services:
                 django_tables_exist=$$?
 
                 if [ $$django_tables_exist -gt 0 ] || [ $$REFRESH_DB == 'true' ]; then
+                    ./cfgov/manage.py update_index
                     ./refresh-data.sh && httpd -d cfgov/apache -D FOREGROUND
```

Pretty sure this command needs to come _after_ the database refresh, or else ES won't have data to index.
```suggestion
                    ./refresh-data.sh && httpd -d cfgov/apache -D FOREGROUND
                    ./cfgov/manage.py update_index
```

---

We might need to rearrang this then to setup the data then load the index and start apache - otherwise the update wouldn't work since the apache server runs in the foreground. Will alter this though.

---

Ohhh, I see. Yeah, I suppose that's what I'd do.